### ISSUE TITLE
php artisan db:seed --class=WorldSeeder Problem

### ISSUE THREAD

**Issue by phemorah** [nnjeim/world#53]
Running the seeder command php artisan db:seed --class=WorldSeeder does nothing. The Countries and States don't get to populate any record in the database.

Kindly assist.. Thank you

**Comment by nnjeim** [comment]
Hi. Please check 2 things1- did you publish the package? php artisan vendor: publish ….etc2- do you any php memory limit issue? Try to set php.ini memory limit to -1Restart your php fpm if using.This should fix the issueThanksSent from my iPhoneOn 17 Apr 2023, at 07:18, Femi Ajao ***@***.***> wrote:﻿
Running the seeder command php artisan db:seed --class=WorldSeeder does nothing. The Countries and States don't get to populate any record in the database.
Kindly assist.. Thank you

**Comment by phemorah** [comment]
Hi, thank you for your prompt response.

There seems to be no luck yet after carrying out your 2 suggestions.

First, I had to go over to roll out the vendor publishing;
php artisan vendor:publish --tag=world once again. And secondly, I set my
php memory_limit under the php.init file from 250M to -1 and had to restart
my local wamp server.

Is there anything else I can do to get the php artisan db:seed
--class=WorldSeeder command populating the countries and states records in
my database please?

Cheers!

**Comment by nnjeim** [comment]
Are you getting any error when trying to seed? I am assuming that you run php artisan migrate.If yes are the tables being created in your DB?Sent from my iPhoneOn 17 Apr 2023, at 13:33, Femi Ajao ***@***.***> wrote:﻿
Hi, thank you for your prompt response.

There seems to be no luck yet after carrying out your 2 suggestions.

First, I had to go over to roll out the vendor publishing;
php artisan vendor:publish --tag=world once again. And secondly, I set my
php memory_limit under the php.init file from 250M to -1 and had to restart
my local wamp server.

Is there anything else I can do to get the php artisan db:seed
--class=WorldSeeder command populating the countries and states records in
my database please?

Cheers!

**Comment by phemorah** [comment]
Yes, I ran a *php migrate" command and the tables were all migrated
successfully.

Meanwhile, I'm not getting any error when I run "php artisan db:seed
--class=WorldSeeder" command. it only shows "seeding database" on the
console when i run the command, it then takes me back to the original
command prompt directory line where I would enter a command again.

**Comment by nnjeim** [comment]
can you please check that your table are created and empty?
if you are running a db:seed and the table are containing data, the seeder action will skip.
as a sanity check i did the following on laravel-world.com
1- I dropped all the existing tables.
2- I have deleted the related entries from the migrations table.
3- I have re-published the package as `php artisan vendor:publish --tag=world --force`
4- I have run the migrations `php artisan migrate`
5- I run the seeded `php artisan db:seed --class=WorldSeeder`
![Screenshot 2023-04-17 at 16 32 32](https://user-images.githubusercontent.com/78922079/232499466-c0cf1bab-375b-470c-ab48-8686d20b19f7.jpg)

If you upgrading from a previous version you are run
`php artisan world:refresh`

**Comment by nnjeim** [comment]
Hi again,

you need to drop the countries, states, cities, languages, currencies and timezone tables. A total of 6.
then edit the migrations table by deleting the rows corresponding to the above tables.

**Comment by nnjeim** [comment]
Hi,

When you require the package nnjeim/world and following the post steps.

you should be able to run the migrate console command.

i have the feeling that there is something wrong in your local setup that is not allowing the migrations in the package to be loaded.

As if i understood you are running in windows it could be related to this.

**Comment by vygandas** [comment]
> if u had the same problem and don't know how to fix it, i fixed it by temporarily increasing the memory limit in the artisan command.
> 
> ```shell
> php -d memory_limit=1G artisan db:seed --class=WorldSeeder
> ```

This helped 🚀
